### ISSUE TITLE
Native JSON body support in expectation file

### ISSUE THREAD

**Issue by drudoi** [mcustiel/phiremock#17]
I propose to support not stings only in request body, but native JSON too. It much easier to define, verify and looks good in IDE with JSON highlight. It's very hard to keep actual big JSON responses if they are already encoded, so I tried to improve it.

In current implementation JSON body is defined in the next way:
```JSON
    "request": {
        "method": "GET",
        "url": {
            "isEqualTo" : "/example_service/some/resource"
        }
    },
    "response": {
        "statusCode": 200,
        "body": "{\"id\": 1, \"description\": \"I am a resource\"}"
    }
```
As you see you should escape quotes and it is not formatted.

I propose to support this definition.
```JSON
    "request": {
        "method": "GET",
        "url": {
            "isEqualTo" : "/example_service/some/resource"
        }
    },
    "response": {
        "statusCode": 200,
        "body": {
            "id": 1, 
            "description": "I am a resource"
        }
    }
```

**Comment by mcustiel** [comment]
Hey, thanks for this proposal. I like the idea. If it doesn't conflict with any other feature I agree with adding it.

**Comment by drudoi** [comment]
@mcustiel, I've added test and finalized the PR.

**Comment by mcustiel** [comment]
Released in v1.6.0. Thank you @dewa1t
